src/insert/spacefinder/rules.ts

```diff
@@ -18,7 +18,10 @@ const MOST_VIEWED_HEIGHT = 600;
 
 const isImmersive = window.guardian.config.page.isImmersive;
 
-const hasImages = !!window.guardian.config.page.lightboxImages?.images.length;
+const hasImagesOrVideo =
+	!!window.guardian.config.page.lightboxImages?.images.length ||
+	window.guardian.config.page.hasYouTubeAtom;
```

Comment:
I think its possible that we have a single video embed rather than a YouTube atom but this will capture the vast majority of cases.